CHANGELOG.md

```diff
@@ -90,6 +90,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - [Dynamic Configurations] Improve dynamic configurations by adding cache and simplifying client fetch ([#6364](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6364))
 - [MD] Add OpenSearch cluster group label to top of single selectable dropdown  ([#6400](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6400))
 - [Workspace] Support workspace in saved objects client in server side. ([#6365](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6365))
+- [Workspace] Add workspaceAccessibility field into application ([#6427](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6427))
```

Comment:
Can we reword this please to make it clearer to indicate what users gain with this change? Maybe something like:
```suggestion
- Allow making apps available in workspaces using `workspaceAccessibility` ([#6427](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6427))
```